frontend/src/components/ViewTrips/trip.js

```diff
@@ -28,12 +41,13 @@ export function getCollaboratorEmails(collaboratorUidArr) {
  * when trips are added and/or editted. Thus, no error checking is done here
  * on the 'display' side.
  *
- * @param {Object} props These are the props for this component:
- * - tripData: Object holding a Trip document fields and corresponding values.
- * - tripId: Document ID for the current Trip document.
- * - handleEditTrip: Handler that displays the edit trip modal.
- * - key: Special React attribute that ensures a new Trip instance is
- *        created whenever this key is updated
+ * @property {Object} props These are the props for this component:
+ * @property {TripData} props.tripData Object holding a Trip document data.
+ * @property {string} props.tripId The document id associated with the trip.
+ * @property {Function} props.handleEditTrip Event handler responsible for
+ *     displaying the edit trip modal.
+ * @property {*} props.key: Special React attribute needed to render the array
```

Comment:
Yeah I was not sure either. But from what I remember seeing I think a key can be any type (number, boolean, etc.) so I think this may be an appropriate use of * here. Let me know if you think otherwise.

Reply:
Well I just found out that key is "not a prop" according to this error message and [this article](https://reactjs.org/warnings/special-props.html):

```
Warning: Trip: `key` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://fb.me/react-special-props)
```

So I think we should remove `key` altogether in the JSDoc for components who technically have one.